admin-guide/upgrade.md

```diff
@@ -0,0 +1,49 @@
+# Upgrade
+
+Upgrade IAM the latest available version is pretty easy.
+
+### Deployment as system daemon
+
+Stop the service:
+```console
+$ systemctl stop iam-login-service
+```
+
+Update the package:
+```console
+$ yum update -y iam-login-service
+```
+
+Restart the service:
+```console
+$ systemctl start iam-login-service
+```
+
+### Deployment as Docker container
+
+Stop and remove the running container:
+```console
+$ docker stop iam-login-service
+$ docker rm iam-login-service
+```
+
+Restart the container with the `latest` tag:
+```console
```

Comment:
Probably, if you already start container with the tag `latest` , you must force an `docker pull indigoiam/iam-login-service:latest`.

Reply:
Sure, I forgot to write it.